common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -278,6 +273,61 @@ public static long byteArrayBaseOffset() {
         return BYTE_ARRAY_BASE_OFFSET;
     }
 
+    private static Data getData() {
+        if (DATA == null) {
+            initialize();
+        }
+        return DATA;
+    }
+
+    private static final Object lock = new Object();
+    private static void initialize() {
+        synchronized (lock) {
+            if (DATA == null) {
+                final boolean useDirectBufferNoCleaner;
+                final AtomicLong directMemoryCounter;
+
+                final long estimateMaxDirectMemory = estimateMaxDirectMemory();
+
+                // Here is how the system property is used:
+                //
+                // * <  0  - Don't use cleaner, and inherit max direct memory from java. In this case the
+                //           "practical max direct memory" would be 2 * max memory as defined by the JDK.
+                // * == 0  - Use cleaner, Netty will not enforce max memory, and instead will defer to JDK.
+                // * >  0  - Don't use cleaner. This will limit Netty's total direct memory
+                //           (note: that JDK's direct memory limit is independent of this).
+                long maxDirectMemory = MaxDirectMemorySetting.get();
+                if (maxDirectMemory >= 0) {
+                    logger.debug("max direct memory was defined using a setter method from the class '{}'",
+                        MaxDirectMemorySetting.class.getName());
+                }
+
+                if (maxDirectMemory < 0) {
+                    maxDirectMemory = estimateMaxDirectMemory;
+                }
+
+                if (maxDirectMemory == 0 || !hasUnsafe() || !PlatformDependent0.hasDirectBufferNoCleanerConstructor()) {
+                    useDirectBufferNoCleaner = false;
```

Comment:
this should be updated to use the new variables names, e.g., `USE_DIRECT_BUFFER_NO_CLEANER`.

Reply:
Same comment as above